static/js/pages/candidate-single.js

```diff
@@ -55,26 +59,51 @@ var electioneeringColumns = [
     className: 'all',
     orderable: true,
     orderSequence: ['desc', 'asc'],
-    render: columnHelpers.buildTotalLink(['electioneering-communications'], function(data, type, row, meta) {
+    render: columnHelpers.buildTotalLink(['electioneering-communications'],
+      function(data, type, row) {
         return {candidate_id: row.candidate_id};
     })
   },
   columns.committeeColumn({data: 'committee', className: 'all'})
 ];
 
-function initFilingsTable() {
-  var $table = $('table[data-type="filing"]');
-  var candidateId = $table.attr('data-candidate');
-  var path = ['candidate', candidateId, 'filings'];
-  tables.DataTable.defer($table, {
-    path: path,
-    columns: filingsColumns,
-    order: [[2, 'desc']],
-    dom: tables.simpleDOM,
-    pagingType: 'simple',
-    hideEmpty: true
-  });
-}
+var itemizedDisbursementColumns = [
+  {
+    data: 'committee_id',
+    className: 'all',
+    orderable: false,
+    render: function(data, type, row) {
+      return columnHelpers.buildEntityLink(
+        row.committee.name,
+        helpers.buildAppUrl(['committee', row.committee_id]),
+        'committee'
+      );
+    }
+  },
+  {
+    data: 'recipient_name',
+    className: 'all',
+    orderable: false,
+    defaultContent: 'NOT REPORTED'
```

Comment:
We haven't been using `defaultContent` elsewhere, so I think best to just leave it blank.